scene/gui/text_edit.h

```diff
@@ -598,6 +598,9 @@ class TextEdit : public Control {
 	void _move_caret_document_start(bool p_select);
 	void _move_caret_document_end(bool p_select);
 
+	// Used in insert_caret_at_carets
+	void _get_new_caret_line_column(int p_old_line, int p_old_wrap_index, int p_old_column, bool p_below, int &p_new_line, int &p_new_column, int p_last_fit_x = -1) const;
```

Comment:
This one's though, bit of a generic name, how about something like `_get_caret_above_below_line_column`?